api/filters.py

```diff
@@ -12,99 +12,115 @@
 
 logger = logging.getLogger(__name__)
 
-NULL_VALUE = 'unspecified'
+NULL_VALUE = "unspecified"
 
 
 class BaseFilter(filters.FilterSet):
     def filter_contains_with_multiple_query_values(self, queryset, name, value):
-        options = set(value.split(','))
+        options = set(value.split(","))
         null_lookup = {}
         if NULL_VALUE in options:
             options.remove(NULL_VALUE)
-            null_lookup = {'__'.join([name, 'isnull']): True}
+            null_lookup = {"__".join([name, "isnull"]): True}
         if options:
             lookup = functools.reduce(
                 operator.or_,
-                {Q(**{'__'.join([name, 'icontains']): item}) for item in options},
+                {Q(**{"__".join([name, "icontains"]): item}) for item in options},
             )
         else:
             lookup = Q(**{})
 
         return queryset.filter(Q(lookup | Q(**null_lookup)))
 
     def filter_exact_with_multiple_query_values(self, queryset, name, value):
-        options = set(value.split(','))
+        options = set(value.split(","))
         null_lookup = {}
         if NULL_VALUE in options:
             options.remove(NULL_VALUE)
-            null_lookup = {'__'.join([name, 'isnull']): True}
-        lookup = {'__'.join([name, 'in']): options}
+            null_lookup = {"__".join([name, "isnull"]): True}
+        lookup = {"__".join([name, "in"]): options}
         return queryset.filter(Q(**lookup) | Q(**null_lookup))
 
 
 class AssetFilter(BaseFilter):
     email = filters.CharFilter(
-        field_name='assigned_to__user__email', lookup_expr='icontains'
+        field_name="assigned_to__user__email", lookup_expr="icontains"
     )
     model_number = filters.CharFilter(
-        field_name='model_number__name',
-        lookup_expr='iexact',
-        method='filter_contains_with_multiple_query_values',
+        field_name="model_number__name",
+        lookup_expr="iexact",
+        method="filter_contains_with_multiple_query_values",
     )
     serial_number = filters.CharFilter(
-        field_name='serial_number',
-        lookup_expr='icontains',
-        method='filter_contains_with_multiple_query_values',
+        field_name="serial_number",
+        lookup_expr="icontains",
+        method="filter_contains_with_multiple_query_values",
     )
     asset_type = filters.CharFilter(
-        field_name='model_number__asset_make__asset_type__name',
-        lookup_expr='iexact',
-        method='filter_contains_with_multiple_query_values',
+        field_name="model_number__asset_make__asset_type__name",
+        lookup_expr="iexact",
```

Comment:
This is better as `icontain` so that a user can useful response values even when not sure of the asset type name or it's spelling. @egichuri